DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/src/kernel_args_restrict.cpp

```diff
@@ -16,8 +16,37 @@ using namespace sycl;
 
 // Forward declare the kernel names in the global scope.
 // This FPGA best practice reduces name mangling in the optimization reports.
-class KernelArgsRestrict;
-class KernelArgsNoRestrict;
+class KernelArgsRestrict_Lambda;
+class KernelArgsNoRestrict_Lambda;
```

Comment:
```suggestion
class KernelArgsRestrict_Lambda;
class Conservative_Lambda;
```

I think instead of KernelArgsRestrict vs KernelArgsNoRestrict, it would be clearer to say KernelArgsRestrict vs Conservative

Reply:
Updated.